README.md

```diff
@@ -1 +1,32 @@
-# landing-page-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# landing-page-review
+
+> This project in the Module-1 understand how to use a Project Review .
+
+![app-screenshot]()
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+to set this project up please run these commands:
+
+`git clone `
+
+`cd `
+
```

Comment:
Please add descriptive README file to your project - please use this [template](https://github.com/microverseinc/readme-template) remember to customize it to your project. It will make your repo professional.